src/main/java/mate/academy/spring/controller/MovieSessionController.java

```diff
@@ -0,0 +1,61 @@
+package mate.academy.spring.controller;
+
+import java.time.LocalDate;
+import java.util.List;
+import java.util.stream.Collectors;
+import mate.academy.spring.model.MovieSession;
+import mate.academy.spring.model.dto.MovieSessionRequestDto;
+import mate.academy.spring.model.dto.MovieSessionResponseDto;
+import mate.academy.spring.service.MovieSessionService;
+import mate.academy.spring.service.mapper.MovieSessionDtoMapper;
+import org.springframework.format.annotation.DateTimeFormat;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class MovieSessionController {
+    private final MovieSessionService movieSessionService;
+    private final MovieSessionDtoMapper movieSessionDtoMapper;
+
+    public MovieSessionController(MovieSessionService movieSessionService,
+                                  MovieSessionDtoMapper movieSessionDtoMapper) {
+        this.movieSessionService = movieSessionService;
+        this.movieSessionDtoMapper = movieSessionDtoMapper;
+    }
+
+    @PostMapping("/movie-sessions")
```

Comment:
```suggestion
    @PostMapping
```

and so on for other methods